### ISSUE TITLE
使用multilabelseg训练后并export的模型应该如何使用PaddleSlim自动压缩？

### ISSUE THREAD

**Issue by hp0716** [PaddlePaddle/PaddleSeg#3926]
### 问题确认 Search before asking

- [x] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### 请提出你的问题 Please ask your question

使用multilabelseg训练得到的模型输出为[1,6,540,856]（NCHW）,使用act压缩工具并沿用pp_mobileseg的config发现输出结果少了的C这个维度，在验证时会出现报错，请问现在是否支持multilabelseg的蒸馏压缩？配置文件该如何修改？
报错如下：

<img width="1717" height="547" alt="Image" src="https://github.com/user-attachments/assets/af61a2f1-436f-4670-8939-4bddd9232337" />
使用的paddle版本如下

<img width="1678" height="116" alt="Image" src="https://github.com/user-attachments/assets/15da6f91-af4a-4e05-b6f3-c3c5dcafb89e" />
目前使用的act配置文件如下

`Global:
  model_dir: /data/code/idcard/api/quality/v6/PaddleSeg/output/pp_mobileseg_idcard_256x256_80k_withaux0715/inference_model_quant
  model_filename: model.pdmodel
  params_filename: model.pdiparams

Distillation:
  alpha: 1.0
  loss: l2
  node:
    - conv2d_75.tmp_0 # change to the name of the output of the last conv in the model.pdmodel in netron

QuantAware:
  onnx_format: False
  quantize_op_types:
    - conv2d
    - depthwise_conv2d

TrainConfig:
  epochs: 20
  eval_iter: 100
  learning_rate: 0.0005
  optimizer_builder:
    optimizer:
      type: SGD
    weight_decay: 4.0e-05
`

**Comment by hp0716** [comment]
上述问题在导出时指定output_op为none即可解决，但出现新的shape不匹配的问题，需要手动修改算子吗？

<img width="1885" height="532" alt="Image" src="https://github.com/user-attachments/assets/8bccbb4a-3c13-4704-9bf6-8e568bfeb87a" />

**Comment by hp0716** [comment]
上述问题已解决，在export.py导出模型时不知道input_shape即可，但蒸馏训练后发现模型文件并未减小，同时loss一直不下降，如下图所示，请问如何解决

<img width="1839" height="1012" alt="Image" src="https://github.com/user-attachments/assets/b4631bb3-847b-4c82-9021-afdd5e19e025" />
